src/validation/tests/property_validation_tests.rs

```diff
@@ -0,0 +1,109 @@
+use plc_index::GlobalContext;
+use plc_source::SourceCode;
+
+use crate::{lowering::validator::ParticipantValidator, test_utils::tests::parse, ErrorFormat};
+
+pub fn lower_and_validate(src: &str) -> String {
```

Comment:
This helper seems to be tailored for validating only the property-lowering. Should we remove the `pub` access-modifier?